src/Math-Operations-Extensions/SequenceableCollection.extension.st

```diff
@@ -0,0 +1,21 @@
+Extension { #name : #SequenceableCollection }
+
+{ #category : #'*Math-Operations-Extensions' }
+SequenceableCollection >> closeTo: aSequenceableCollection [
+	"Return true if all my elements are close to the elements of same index of the parameter"
+
+	"#(1.9283901234902349 2.302949083493849) closeTo: #(1.9283901234902348 2.302949083493899) >>> true"
+
+	self with: aSequenceableCollection do: [ :a :b | (a closeTo: b) ifFalse: [ ^ false ] ].
```

Comment:
code duplication. 
Could use 
```smalltalk
self closeTo: aSequenceableCollection precision: Float closeToPrecision
``` 
where `#closeToPrecision` will return `0.0001` (extracted from `Float>>#closeTo:`)

Reply:
Oh good catch :) I’ll update the PR